test/semanticValidation/specification/validateOperation/pathParameterDefinition.json

```diff
@@ -0,0 +1,96 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure File Storage",
+    "version": "2021-04-10",
+    "x-ms-code-generation-settings": {
+      "header": "MIT",
+      "strictSpecAdherence": false
+    }
+  },
+  "securityDefinitions": {
+    "File_shared_key": {
+      "type": "apiKey",
+      "name": "Authorization",
+      "in": "header"
+    }
+  },
+  "schemes": ["https"],
+  "consumes": ["application/xml"],
+  "produces": ["application/xml"],
+  "paths": {},
+  "x-ms-paths": {
+    "/{shareName}?restype=share": {
```

Comment:
Can you add more coverage to this swagger i.e. more paths with path parameter defined in operation level, and operation without path parameter, and operation with path parameter referenced from globl level, etc.